src/commands/memory-stats.json

```diff
@@ -11,6 +11,111 @@
             "NONDETERMINISTIC_OUTPUT",
             "REQUEST_POLICY:ALL_SHARDS",
             "RESPONSE_POLICY:SPECIAL"
-        ]
+        ],
+        "reply_schema": {
+            "description": "memory usage details",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "peak.allocated": {
+                    "type": "integer"
+                },
+                "total.allocated": {
+                    "type": "integer"
+                },
+                "startup.allocated": {
+                    "type": "integer"
+                },
+                "replication.backlog": {
+                    "type": "integer"
+                },
```

Comment:
i think that MEMORY STATS shouldn't document which fields it has, and leave it free for new ones to be added.

Reply:
i'm not sure we wanna list (document) the fields of this command.
and if we don't, i'd like to put a comment here that we omitted that on purpose, so someone won't try to add them.